apps/etl-func/src/adapters/env.ts

```diff
@@ -1,10 +1,13 @@
 import { z } from "zod";
 
 export const envSchema = z.object({
-  EVENTHUB_CONNECTION_URI: z.string().url(),
+  COSMOSDB_DATABASE_NAME: z.string().min(1),
+  COSMOSDB_URI__accountEndpoint: z.string().url(),
+  EVENTHUB_CONNECTION_URI: z.string(),
```

Comment:
I would use a specific name for this event hub because we are going to create another event hub for message-status.
```suggestion
  MESSAGE_EVENTHUB_CONNECTION_URI: z.string(),
```

Reply:
EVENTHUB_CONNECTION_URI is the eventhub namespace FQDN.
If we are going to use the same namespace, and i think this is probably the case for message-status, this remains the same and if you want we can call it EVENTHUB_NAMESPACE_CONNECTION_URI instead (but doesnt sound good to me).

If you mean that we are going to interact with multiple eventhub namespaces in the future than i agree with you and we must specialized this one like MESSAGES:EVENTHUB_CONNECTION_URI

what do you think ?

Reply:
You are right @lucamancosu